CC/Sounder/include/BaseRadioSetUHD.h

```diff
@@ -0,0 +1,59 @@
+#include <SoapySDR/Formats.hpp>
+#include <SoapySDR/Time.hpp>
+#include <uhd/usrp/multi_usrp.hpp>
+
+#include "config.h"
+#include "nlohmann/json.hpp"
+
+#ifndef BASE_RADIO_SET_UHD_H
+#define BASE_RADIO_SET_UHD_H
+
+class RadioUHD;
```

Comment:
Remove forward declarations

Reply:
the use of the forward declarations is what the current Sounder has for the Base/ClientRadioSet.h. 
Do you recommend replacing this with #include "RadioUHD.h" or keep consistent with what currently had in the develop branch